### PR TITLE
Pipeline

### PR DESCRIPTION
**Pull request by yongzhi1**

### CODE REVIEW

File: tools/topology/sof/pipe-kfbm-capture.m4 [hunk 1]
```diff
@@ -2,7 +2,7 @@
 #
 # Pipeline Endpoints for connection are :-
 #
-#  host PCM_C <-+- KPBM 0 <-- B1 <-- Volume 0 <-- B0 <-- source DAI0
+#  host PCM_C <-+- KPBM 0 <-- B1 <-- B0 <-- source DAI0
```
Comment (ranj063) [inline tools/topology/sof/pipe-kfbm-capture.m4]: This doesnt sound right. AFAIK SOF doesnt allow connecting 2 buffers?

Comment (yongzhi1) [reply tools/topology/sof/pipe-kfbm-capture.m4]: The Volume has been removed, this is just to update the description to match the topology update 078c3c1a.

Comment (ranj063) [reply tools/topology/sof/pipe-kfbm-capture.m4]: @yongzhi1 volume couldnt have been removed with removing a buffer too. the connecttion B1 <-- B0 is not possible

Comment (yongzhi1) [reply tools/topology/sof/pipe-kfbm-capture.m4]: you are right,  actually line 70 of original file already updated:

host PCM_C <-- B1 <--+- KPBM 0 <-- B0 <-- source DAI0

**Pull request merged**